infra/common/src/test/java/cn/hippo4j/common/executor/ThreadFactoryBuilderTest

```diff
@@ -0,0 +1,83 @@
+package cn.hippo4j.common.design.builder;
+/*
```

Comment:
Add the open source agreement to the top, and leave a blank line with package

Reply:
ok